control-plane/plugin/src/resources/mod.rs

```diff
@@ -74,6 +74,16 @@ pub enum ScaleResources {
     },
 }
 
+/// The types of resources that support the 'expand' operation.
+#[derive(clap::Subcommand, Debug)]
+pub enum ExpandResources {
+    /// Expand Pool.
```

Comment:
Something like this, you could elaborate

```suggestion
    /// Expand a Pool to cover the entire span of the disk.
    /// Pool is only expandable if it has been created with sufficient expandable slack via `maxExpansion`.
```

Reply:
Pool expansion is possible to certain extent without `max_expansion` also. As cluster size increases with default max_expansion also user will see higher `max_expandable_size`.


/// Expand a Pool to cover the entire span of the disk.
/// Please resize underlying disk before executing this cmd.
/// Refer `max_expandable_size` of the pool before resizing disk.

Reply:
Please also include the bit about creating the Pool with sufficient slack via `maxExpansion` since that's very important

Reply:
Is it needed here? Its well documented on CRD schema docs.

Reply:
User may not even read the CRD schema
Since this is a tricky operation, I think we better have good docs at any user endpoint

Reply:
/// Please refer maxExpansion parameter in the docs which controls maxExpandableSize.